substra/cli/interface.py

```diff
@@ -143,11 +168,21 @@ def click_option_verbose(f):
     """Add verbose option to command."""
     return click.option(
         '--verbose',
+        expose_value=False,
+        callback=save_to_global_conf,
         is_flag=True,
         help='Enable verbose mode.'
     )(f)
 
 
+def click_option_common(f):
```

Comment:
I'll rename it yes!

I'm not sure about adding `output_format`, because it means that we will add this option even where it is not useful (for example in `describe` or `download`) as we did for expand. 🤔 
So removing them from the `GlobalConf` looks better to me, what do you think about that?

Reply:
Has been discussed directly.